spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -8,7 +8,7 @@
                             :language, :identifier, :based_near, :related_url] }
   end
 
-  let(:collection) { Collection.new(description: 'a nice collection', title: 'A clever title') }
+  let(:collection) { Collection.new(id: '111', description: 'a nice collection', title: 'A clever title') }
```

Comment:
The solr query now uses the ID to find the size.  The solr query fails without the id